spec/api-session-spec.js

```diff
@@ -289,12 +289,19 @@ describe('session module', () => {
       res.end(mockPDF)
       downloadServer.close()
     })
+
+    const isPathEqual = (path1, path2) => {
+      const absPath1 = path.resolve(path1)
+      const absPath2 = path.resolve(path2)
```

Comment:
I'm not sure that we need to `resolve()` these before passing them to `relative()` -- it looks like [`relative()`](https://github.com/nodejs/node/blob/master/lib/path.js) does that itself, so this effectively just calls it twice